studies/migrations/0071_auto_20210629_1434.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.0.14 on 2021-06-29 18:34
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("studies", "0070_auto_20210521_0632"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="response",
+            options={
+                "base_manager_name": "related_manager",
+                "permissions": (
+                    (
+                        "view_all_response_data_in_analytics",
+                        "View all response data in analytics",
+                    ),
+                ),
+            },
+        ),
+    ]
```

Comment:
Before I get too far into this I have to admit I still need to do some research on this.  Basically, there are existing migrations to add `ordering` in the meta class using the `AlterModelOptions` class.  This can be seen [here](https://github.com/lookit/lookit-api/blob/e390e5fc536b9d0a47a976184a469791a3e51f25/studies/migrations/0017_auto_20170710_1449.py#L15-L21) and in other migrations.  What I don't know is "why?".  

To answer your other two questions, yes and yes.

Reply:
ah-hah, I see the absence of "ordering" under options is actually what's removing it here. I was expecting to see the removed thing named like if we were removing a field. Sounds good!